### PR TITLE
Add Getting Started section to docs

### PR DESCRIPTION
**Pull request by marklewin**
## Description

[DEVX-3143](https://nexmoinc.atlassian.net/browse/DEVX-3143)

This is an early draft PR with some placeholder content just so we can agree the layout and what topics to include.

I've used the tutorial format for this (because step-by-step seemed the best way to present this info) but annoyingly this gives us a section that contains a Tutorials subfolder with a single tutorial in it. It would be nice to see this at the top level IMO, but I'm not sure this is even possible. Each of the tutorial steps outlines what I'd like to cover.

I'd love to get your input on both the format and suggested content before I develop this further. Thanks :)

## Review

* [Page to review](https://nexmo-develo-markl-gett-71j9iu.herokuapp.com/getting-started/overview)

### CODE REVIEW

**Comment by tbedford** [comment]
I think the nav spec is expecting Concepts at 0 but we can change the spec. Not looked in detail though.

**Comment by marklewin** [comment]
> Re: build failure. I think the nav spec is expecting Concepts at 0 but we can change the spec. Not looked in detail though.

Thanks - had a go at doing that!

**Comment by fabianrbz** [comment]
@marklewin I ❤️ the idea, and I do think that we can get rid of the annoying `tutorials` section,  I'll need to look into it though

File: _documentation/en/getting-started/overview.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+- [Signing up for an account](#signing-up-for-an-account)
+- [Accessing the Developer Dashboard](#accessing-the-developer-dashboard)
+- [Using the Nexmo CLI](#using-the-nexmo-cli)
+- [Experimenting with our APIs](#experimenting-with-our-apis)
+- [Using a REST API client library](#using-a-rest-api-client-library)
+- [Working with Webhooks](#working-with-webhooks)
+- [What to do next](#what-to-do-next)
+
+## Signing up for an account
```
Comment (tbedford) [inline _documentation/en/getting-started/overview.md:25]: Markdown Linter tells me we should always have a blank line after a heading. I've usually adopted that approach.

File: _documentation/en/getting-started/overview.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+* **View your API key and secret**. You will need these to authenticate your requests to our APIs
+* **Manage your account balance**. Access to our APIs is charged on a  per-request basis. We’ll give you some free credit when you first open your account and you can [top up](/numbers/guides/payments) when you run low.
+* **Rent virtual numbers**. You can use [virtual numbers](/concepts/guides/glossary#virtual-number) provided by Vonage to send and receive calls and messages. See [rent a virtual number](/numbers/guides/number-management#rent-a-virtual-number).
+* **Manage applications**. Some of our APIs (such as Voice and Messages) require you to create an [Application](/application/overview), which acts as a container for security and configuration information. You can create and manage these applications in the Developer Dashboard.
+* **Manage your account**. You can perform [other administration tasks](/account/guides/dashboard-management) here.
+
+## Using the Nexmo CLI
+The Developer Dashboard is a GUI-based approach to managing your account. You can optionally perform many of the same tasks from the command line, using the Nexmo CLI.
```
Comment (tbedford) [inline _documentation/en/getting-started/overview.md:38]: > The Developer Dashboard is a GUI-based approach to managing your account.

Would actually be quite nice to have this sentence in the previous Dashboard section, just to orient users as to what Dashboard is. You could then go on to say something along the lines of - As an alternative to the GUI approach to managing your account described in the previous section (link to previous section) there is also...blah blah...

File: _documentation/en/getting-started/overview.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+This is often quicker and also allows you to script these operations.
+The Nexmo CLI is written in Node.js and installed using the Node Package Manager (`npm`). The `README` in the [Nexmo CLI GitHub repo](https://github.com/Nexmo/nexmo-cli) shows you how to install and work with the Nexmo CLI.
+
+## Experimenting with our APIs
+Ultimately, you’re going to want to build an app in your [chosen programming language](#using-a-rest-api-client-library). But to start with, you might want to make some sample requests to our APIs to check that you have provided the correct parameters and that you are getting back the responses you want.
+If you are familiar with the command-line tool [Curl](https://curl.haxx.se/), you will find Curl snippets for each of our APIs that you can copy, paste and modify.
+However, various GUI tools exist that make this a less painful experience. A popular one is [Postman](https://www.postman.com/). Read our guide on [using Postman to work with our APIs](/tools/postman).
+
+## Using a REST API client library
```
Comment (tbedford) [inline _documentation/en/getting-started/overview.md:47]: Client Libraries are now referred to as Server SDKs in the docs.

File: _documentation/en/getting-started/overview.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+- [Java](https://github.com/nexmo/nexmo-java)
+- [.NET](https://github.com/nexmo/nexmo-dotnet)
+- [PHP](https://github.com/nexmo/nexmo-php)
+- [Python](https://github.com/nexmo/nexmo-python)
+- [Ruby](https://github.com/nexmo/nexmo-ruby)
+
+Click on the link for your chosen language to visit that library’s GitHub repo page, where you can learn how to install and use it.
+
+## Working with Webhooks
+Once you have learned how to make requests to our APIs to place calls, send messages, etc. you will want to learn how to receive inbound communications on your virtual number.
```
Comment (tbedford) [inline _documentation/en/getting-started/overview.md:60]: Avoid 'etc.' - use 'and so on', 'for example', or something else.

File: _documentation/en/getting-started/overview.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+Visit our guide on [testing with Ngrok](https://developer.nexmo.com/tools/ngrok) to learn how to use it.
+
+## What to do next
+Once you have understood the basic concepts and installed the tools you intend to use, you are ready to start building! View the documentation for your chosen API to get started.
+
+If you encounter any difficulties, check out our knowledge base and reach out for assistance if necessary at [our support site](https://help.nexmo.com/).
+
+We take pride in our documentation, but are always looking to make it better. So if you find anything that is unclear or lacking the information you require, then please submit feedback for the topic in question. If we’ve done a good job on a particular section then please let us know!
+
+
```
Comment (tbedford) [inline _documentation/en/getting-started/overview.md:77]: Just a suggestion, but it might be useful to add a list of links on the end to perhaps some of the more obvious Next Steps (such as things already linked to in the Getting Started, and perhaps one or two simple tasks like Send a Unicode SMS).

Comment (marklewin) [reply _documentation/en/getting-started/overview.md:77]: Good idea!

File: config/locales/en.yml [hunk 6]
```diff
@@ -83,56 +82,68 @@ en:
...
-    once_you_have_the_cli_installed_html: >-
-      Once you have the CLI installed you can use it to create a Nexmo application. Run the following command and make a note of the application ID that it returns. This is the value to use in <code>NEXMO_APPLICATION_ID</code> in the example below. It will also create <code>private.key</code> in the current directory which you will need in the <em>Initialize your dependencies</em> step
-    nexmo_needs_to_connect_html: >-
-      Nexmo needs to connect to your local machine to access your <code>answer_url</code>. We recommend using <a href="https://www.nexmo.com/blog/2017/07/04/local-development-nexmo-ngrok-tunnel-dr/">ngrok</a> to do this. Make sure to change <code>demo.ngrok.io</code> in the examples below to your own ngrok URL.
+    nexmo_application_contains_html: A Nexmo application contains the required configuration
```
Comment (tbedford) [inline config/locales/en.yml:90]: Not completely sure why these changes are part of Getting Started?

File: config/redirects.yml [hunk 7]
```diff
@@ -162,7 +162,7 @@
 "/vonage-business-cloud/number-programmability/guides/vbc-resources": "/vonage-business-cloud/smart-numbers/guides/vbc-resources"
 "/vonage-business-cloud/number-programmability/code-snippets/vbc-receive-inbound-call": "/vonage-business-cloud/smart-numbers/code-snippets/vbc-receive-inbound-call"
 "/vonage-business-cloud/number-programmability/code-snippets/vbc-route-to-extension": "/vonage-business-cloud/smart-numbers/code-snippets/vbc-route-to-extension"
-"/tutorials/client-sdk-ios-make-receive-calls-swift": "https://developer.nexmo.com/client-sdk/in-app-voice/guides/start-and-receive-calls/ios"
+"/tutorials/client-sdk-ios-make-receive-calls-swift": https://developer.nexmo.com/client-sdk/in-app-voice/guides/start-and-receive-calls/ios
```
Comment (tbedford) [inline config/redirects.yml:165]: This looks like an error here as quotes were removed.

**Comment by marklewin** [comment]
> What is the `.keep` file?

That was committed by Fabian to preserve an empty folder (which isn't recognised by GitHub). The folder is no longer empty, so I will remove it.

File: config/redirects.yml (hunk 7 above)
Comment (marklewin) [reply config/redirects.yml:165]: Committed in error and reverted

File: config/locales/en.yml (hunk 6 above)
Comment (marklewin) [reply config/locales/en.yml:90]: Committed in error and reverted

File: _documentation/en/getting-started/overview.md [hunk 8]
```diff
@@ -0,0 +1,88 @@
...
+These endpoints are called Webhooks. [Find out more about webhooks](/concepts/guides/webhooks). Once you have created your web hook, you must tell our API platform to use it. The process for doing this depends on which API you are using and full instructions can be found in the documentation for that API.
+
+Making these webhooks publicly-accessible during development can be tricky, so we recommend a tool called [Ngrok](https://ngrok.com/).
+
+Visit our guide on [testing with Ngrok](https://developer.nexmo.com/tools/ngrok) to learn how to use it.
+
+## What to do next
+
+Once you have understood the basic concepts and installed the tools you intend to use, you are ready to start building! View the documentation for your chosen API to get started, or try out some of the following simple tasks:
```
Comment (tbedford) [inline _documentation/en/getting-started/overview.md:78]: I think both 'concepts' and 'tools' could link to our docs. We have a concepts page and a tools page.

Comment (marklewin) [reply _documentation/en/getting-started/overview.md:78]: I'm not sure ... the concepts and tools here specifically refer to the concepts and tools covered in this getting started guide. I don't want to distract them :)  I'll link out to these separately: see if you think this works?
